libraries/botbuilder-adapter-twilio-whatsapp/src/twilioWhatsAppAdapter.ts

```diff
@@ -1,12 +1,24 @@
-import { Activity, ActivityTypes, BotAdapter, TurnContext, ConversationReference, ResourceResponse, WebRequest, WebResponse } from 'botbuilder';
+import { Activity, ActivityTypes, BotAdapter, TurnContext, ConversationReference, ResourceResponse, WebRequest, WebResponse, TokenResponse } from 'botbuilder';
 import * as Twilio from 'twilio';
 import { MessageInstance } from 'twilio/lib/rest/api/v2010/account/message';
 import { parse } from 'qs';
+import {
+    AppCredentials,
+    JwtTokenValidation,
+    MicrosoftAppCredentials,
+    SimpleCredentialProvider,
+    TokenApiClient,
+    TokenApiModels
+} from 'botframework-connector';
+import { USER_AGENT } from 'botbuilder/src/botFrameworkAdapter';
```

Comment:
Does this work for you? I receive an error while testing in another module. 

`Error: Cannot find module 'botbuilder/src/botFrameworkAdapter'`

Reply:
In previous versions to 4.6 of the botbuilder package it's not yet exported. That's one of the reasons I updated the version.